powers.c

```diff
@@ -0,0 +1,13 @@
+#include <stdio.h>
+/*The purpose is to find all the values of 2^n under 1000. Jack Rosen*/
+int main()
+{
+	int product = 1;
+	printf("%d\n",product);
+	while (product <= 500)
```

Comment:
this is a correct implementation, but what if you didn't know the final number beforehand? What if we asked you to do up to 10,000? 1000,000? You'd have to create a customized code for each one.